examples/notebooks/lambdify.ipynb

```diff
@@ -0,0 +1,810 @@
+      "\n",
+      "We will investigate the effect of `lambdify` function with `numpy` module on the speed of numerical computation  and compare it with computation in SymPy natively. \n",
+      "\n",
+      "**All the tests are run on quadcore Intel\u00ae Core\u2122 i5-3210M CPU @ 2.50GHz on a 64-bit Ubuntu 14.04 installation**\n",
+      "\n",
+      "\n",
+      "\n",
+      "   "
+     ]
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "\n",
+      "##Using the `lambdify()` function. "
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "from IPython.display import display\n",
+      "from sympy.interactive import printing\n",
+      "printing.init_printing(use_latex='mathjax')\n",
+      "from __future__ import division\n",
+      "import sympy as sym\n",
+      "from sympy import *\n",
+      "import numpy as np\n",
+      "from numpy import *"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": [],
+     "prompt_number": 1
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "\n",
+      "\n",
+      "Defining a SymPy symbol `xs`"
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "xs = symbols('x')"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": [],
+     "prompt_number": 2
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "Creating a symbolic function fn"
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "fn = xs**2\n",
+      "fn"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": [
+      {
+       "latex": [
+        "$$x^{2}$$"
+       ],
+       "metadata": {},
+       "output_type": "pyout",
+       "prompt_number": 3,
+       "text": [
+        " 2\n",
+        "x "
+       ]
+      }
+     ],
+     "prompt_number": 3
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "Now using the `lambdified` function on fn"
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "fn_lambdified = lambdify(xs,xs**2)"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": [],
+     "prompt_number": 4
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "Using `numpy` modules `linspace` to create an array x with 1000 datapoints between 0 & 1"
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "x = np.linspace(0,1,1000)"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": [],
+     "prompt_number": 5
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "Using IPython Notebook command **`%timeit`** to record the time taken to run the command"
```

Comment:
The `% timeit` measures wall time by default [1], wall time is the time you see that has passed. Wall time can vary even if you use the same computer to run the same code, depending what other programs are also running in the background. You should use `CPU time` for this use case.

[1]: ` https://docs.python.org/2/library/timeit.html#timeit.default_timer